internal/flink/command_statement.go

```diff
@@ -32,6 +32,7 @@ func (c *command) newStatementCommand() *cobra.Command {
 	cmd.AddCommand(c.newStatementExceptionCommand())
 	cmd.AddCommand(c.newStatementListCommand())
 	cmd.AddCommand(c.newStatementStopCommand())
+	cmd.AddCommand(c.newStatementResumeCommand())
```

Comment:
```suggestion
	cmd.AddCommand(c.newStatementResumeCommand())
	cmd.AddCommand(c.newStatementStopCommand())
```
Nitpick: Alphabetical order